explorer/src/server/rpcServer.ts

```diff
@@ -0,0 +1,20 @@
+import jayson from 'jayson'
+import * as rpcMethods from './rpcMethods'
+
+const serverOptions = {
+  useContext: true, // permits passing extra data object to RPC methods as 'server context'
+}
+
+export const rpcServer = new jayson.Server(rpcMethods, serverOptions)
+
+// broken typing for server.call - should be able to accept 3 arguments
+// https://github.com/tedeh/jayson#server-context
+// https://github.com/tedeh/jayson/pull/152
```

Comment:
How do you feel about opening an issue for them to publish a new version with this fix?

Reply:
@rupurt so that change has already been merged to master, they just haven't made a new release yet. Should I raise an issue just asking for the release?

Reply:
@RyanRHall I feel like we should. It hasn't had a release in a while

Reply:
Okay! I'll make that request. Thanks.